contracts/BoostedEthBridgeSwapper.sol

```diff
@@ -60,7 +60,7 @@ contract BoostedEthBridgeSwapper is ZkSyncBridgeSwapper {
         }
     }
 
-    function swapEthForYvCrv(uint256 _amountIn) internal returns (uint256) {
+    function swapEthForYvCrv(uint256 _amountIn) public payable returns (uint256) {
```

Comment:
Because sometimes for operational reasons it's convenient to get yvCrvStETH in a single call, without having it go on the L2 automatically. We discussed this at the time

Reply:
I think that it is not a good idea. We're creating multiple entry points to the contract. And we are breaking the assumption that the methods don't need authorisation since the funds will end up in the L2. For example one could call that method before we call `exchange` and prevent us from bringing liquidity to the L2.

I understand why it helps for testing, deployment, etc, but it changes the security model of the flow. We should keep that method internal.